Cargo.toml

```diff
@@ -60,3 +69,6 @@ incremental = false
 debug-assertions = false
 overflow-checks = false
 rpath = false
+
+[lib]
+bench = false
```

Comment:
why

Reply:
Because  `--output-format` is an argument that Criterion.rs supports but libtest does not.

According to https://bheisler.github.io/criterion.rs/book/faq.html#cargo-bench-gives-unrecognized-option-errors-for-valid-command-line-options, we could either disable benchmarks for crate or run only Criterion benchmark.

Since I have used the latter method due to only benching `benches_badger_rocks`, I will restore these modifications on Cargo.toml.